symfony/client/src/orangehrmAuthenticationPlugin/pages/AdministratorAccess.vue

```diff
@@ -28,15 +28,16 @@
         @submitValid="onSubmit"
       >
         <oxd-text tag="h6" class="orangehrm-admin-access-title">
-          Administrator Access
+          {{ $t('general.admin_access') }}
         </oxd-text>
 
         <br />
 
-        <oxd-text :class="noteClasses">
-          You have requested to access a critical Administrator function in
-          OrangeHRM and are required to validate your credentials below
-        </oxd-text>
+        <div :class="noteContainerClass">
+          <oxd-text tag="toast-message">
+            {{ $t('general.admin_access_note') }}
+          </oxd-text>
+        </div>
```

Comment:
https://github.com/orangehrm/orangehrm/pull/1171#discussion_r828066269
Couldn't find a suitable tag for the title  (font-weight) that had both needed size and weight